server/src/main/java/io/pravega/schemaregistry/storage/client/TableStore.java

```diff
@@ -178,18 +176,19 @@ protected void doStop() {
     }
 
     public CompletableFuture<List<Version>> updateEntries(String tableName, Map<byte[], VersionedRecord<byte[]>> batch) {
```

Comment:
Please add a @NonNull on the batch to assert the assumption here (although I can see from the caller code that it cannot be null). Since this is a public method input validation for `tableName` would be useful too.

Reply:
added precondition